setup.py

```diff
@@ -2,19 +2,42 @@
 
 from setuptools import setup
 
+import i18n
```

Comment:
Right, that's the kind of fragility I'm talking about.  And the next developer could add an import to `__init__.py` without understanding the need to avoid importing third-party libraries, even transitively, into there, or the installation will fail.